jenkins/freestyle/delete-old-workers.groovy

```diff
@@ -0,0 +1,19 @@
+import hudson.model.*
+import hudson.node_monitors.*
+import hudson.slaves.*
+import java.util.concurrent.*
+
+import hudson.model.*;
```

Comment:
When running this as a job, it [looks like it needs these statements](https://build.testeng.edx.org/view/All/job/clean-up-workers/3/console).